frappe/geo/languages.csv

```diff
@@ -0,0 +1,78 @@
+language_code,language_name,enabled
+af,Afrikaans,0
+am,አማርኛ,0
+ar,العربية,1

```

Comment:
In case you also see the fields in a wrong order, that's a rendering bug (probably due to the language being right to left):

![Bildschirmfoto 2024-03-27 um 00 12 26](https://github.com/frappe/frappe/assets/14891507/4c61866e-01ad-4c2c-aa20-dd795426802e)

The underlying data is still in the correct order.